### PR TITLE
Update pihole-google.txt

### PR DESCRIPTION
**Pull request by gallegonovato**

### CODE REVIEW

File: pihole-google.txt [hunk 1]
```diff
@@ -7452,4 +7462,345 @@ www.tf-enterprise-devel.gcp.fitbit.com
 www.vps.store-au.fitbit.com
 www.wiki.fitbit.com
 www.windchill.fitbit.com
-wwwinternal.fitbit.com
\ No newline at end of file
+wwwinternal.fitbit.com
+ 1e100.net       # Google Cookies
```
Comment (nickspaargaren) [inline pihole-google.txt:7466]: Can you please remove all comments? The automatic will probably fail because of the extra comments. Thank you!

**Comment by nickspaargaren** [comment]
The "Duplicate checker" is checking for duplicate lines. If there are one or more duplicates I won't be able to directly merge te pull request. The Red Cross will be shown. If there are no duplicates, a green check mark will be present.

In the future it is best to open pull requests with as little domains possible. For example by category. This is also way easier to review the addition, thank you and a happy new year, you're amazing!

File: pihole-google.txt (hunk 1 above)
Comment (gallegonovato) [reply pihole-google.txt:7466]: I'm trying but it won't let me. if I close this suggestion, and reopen it, are things saved or removed?

Comment (nickspaargaren) [reply pihole-google.txt:7466]: If you resolve and reopen this suggestion/review. No changes will be made to your files.

**Pull request closed without merging**